CHANGELOG.md

```diff
@@ -2,4 +2,6 @@
 
 * [CHANGE] Removed global metrics for KV package. Making a KV object will now require a prometheus registerer that will
   be used to register all relevant KV class metrics. #22
-* [CHANGE] Added CHANGELOG.md and Pull Request template to reference the changelog
\ No newline at end of file
+* [CHANGE] Added CHANGELOG.md and Pull Request template to reference the changelog
+* [ENHANCEMENT] Add `dslog` package #
```

Comment:
I kind of like the look of this, do you think we should do this on all of these changes going forward? We could also add a new tag like [Addition] or something if we'd like.

Reply:
@treid314 I'm actually not 100% sure, I just thought it was the right thing to do since @pracucci asked us to keep the changelog up to date.

Reply:
I've mostly been using the changelog to track any constructor or method input changes, there just haven't really been any for a bit. I think it's great to track adding modules as well to show us whats in here.